modules/tensor_mechanics/doc/content/modules/tensor_mechanics/beam_vandv.md

```diff
@@ -0,0 +1,254 @@
+
+
+!media media/tensor_mechanics/vandv/euler_finite_rot_x2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:euler_finite_rot_x
+       caption=Displacement of the cantilever beam with large strain and rotation in X direction
+
+
+!media media/tensor_mechanics/vandv/euler_finite_rot_y2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:euler_finite_rot_y
+       caption=Displacement of the cantilever beam with large strain and rotation in Y direction
+
+
+## Torsion
+
+A 1D cantilever beam of 1 m is modeled using beam elements in MOOSE. The beam has a cross section area of 0.5 ${m}^2$, moment of inertia about Y and Z axis of 1e-5 ${m}^4$, Young's modulus of elasticity of 2e9 N/${m}^2$, poisson's ratio of 0.3 and shear coefficient of 1. A torsion of magnitude 5 N/m is applied at the free end of the cantilever beam.
+
+!listing torsion_1.i
+
+### Results
+
+The analytical solution for the axial twist at the free end of the beam is given by:
+\begin{equation}
+\phi = \frac {TL}{GI_{x}} = 3.25e-4 rad
+\end{equation}
+\begin{equation}
+where, I_{x} = I_{y} + I_{z} = 2e-5 m^4
+\end{equation}
+
+The rotations at the free end of the beam obtained in MOOSE is 3.2e-4 rad as shown in [fig:torsion2]
+
+!media media/tensor_mechanics/vandv/torsion2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:torsion2
+       caption=Rotation of the beam due to the applied torsion.
+
+## Small strain Euler beam vibration
+
+A 1D cantilever beam of 4 m is modeled using beam elements in MOOSE. The beam has a cross section area of 0.01 ${m}^2$, moment of inertia about Y and Z axis of 1e-4 ${m}^4$, Young's modulus of elasticity of 1e4 N/${m}^2$, poisson's ratio of -0.99, density of 1 kg/${m}^3$, shear modulus of 4e7 N/${m}^2$ and shear coefficient of 1. An impulse load with a peak load of 0.01 N at 0.05 s as shown in [fig:impulseload] is applied at the free end of the beam in Y direction. The Newmark time integration parameters used in the problem corresponds to the Newmark's average acceleration method, i.e `beta=0.25` and `gamma=0.5`.
+
+!media media/tensor_mechanics/vandv/euler_small_impulse.png
+       style=width:500px;margin-left:110px;float:center;
+       id=fig:impulseload
+       caption=Impulse load pattern applied at the free end of the cantilever beam.
+
+!listing dyn_euler_small.i
+
+### Results
+
+For this beam, the dimensionless parameter alpha is given by:
+\begin{equation}
+alpha = \frac {kAGL^2}{EI} = 6.4e6
+\end{equation}
+ Therefore, the beam behaves like a Euler-Bernoulli beam. The displacement, velocity and acceleration as the function of time, at the free end of the cantilever beam in MOOSE is compared with the results from ABAQUS using the time step of 0.05 s as shown in [fig:euler_vibration_disp], [fig:euler_vibration_vel] and [fig:euler_vibration_acc].
```

Comment:
Subject-verb agreement here too: the displacement, velocity, and acceleration....are compared.. Note this problem shows up in many of the results sections on this page. If you fix this paragraph, please fix the others as well.